test/viewmodels/SecuritiesRequestedStatementsViewModelSpec.scala

```diff
@@ -0,0 +1,207 @@
+import utils.Utils.{emptyString, spanComponent}
+
+import java.time.LocalDate
+
+class SecuritiesRequestedStatementsViewModelSpec extends SpecBaseWithSetup {
+
+  "prepareStatementRows" should {
+    "return a sequence of StatementRow" in {
+      val viewModel = createViewModel(securityStatements)
+      val result = viewModel.statementRows
+
+      result mustBe Seq(
+        StatementRow(
+          historyIndex = 0,
+          eori = None,
+          date = "10 July 2023 to 20 July 2023",
+          pdf = None,
+          rowId = "requested-statements-list-0-row-0",
+          dateCellId = "requested-statements-list-0-row-0-date-cell",
+          linkCellId = "requested-statements-list-0-row-0-link-cell"
+        )
+      )
+    }
+
+    "return a sequence of StatementRow with correct data" in {
+      val securityStatements = Seq(
+        SecurityStatementsForEori(
+          eoriHistory = EoriHistory("EORI456", Some(startDateJuly), Some(endDateJuly)),
+          currentStatements = Seq(requestedStatement),
+          requestedStatements = Seq(requestedStatement)
+        ),
+        SecurityStatementsForEori(
+          eoriHistory = EoriHistory("EORI789", Some(startDateJune), Some(endDateJune)),
+          currentStatements = Seq(requestedStatement),
+          requestedStatements = Seq(requestedStatement)
+        )
+      )
+
+      val viewModel = createViewModel(securityStatements)
+      val result = viewModel.statementRows
+
+      result.size mustBe 2
+      result.head.eori mustBe None
+      result(1).eori mustBe Some("EORI789")
+    }
+  }
+
+  "renderEoriHeading" should {
+    "render the EORI" in {
+      val row = StatementRow(
+        0,
+        Some(securityStatementForEori.eoriHistory.eori),
+        "10 July 2023 to 20 July 2023",
+        Some(pdfLink),
+        "rowId",
+        "dateCellId",
+        "linkCellId"
+      )
+
+      val result = row.eori
+
+      result mustBe Some(securityStatementForEori.eoriHistory.eori)
+
+      val expectedSize = securityStatementForEori.eoriHistory.eori.length
+      val actualSize = result.map(_.length)
+
+      actualSize mustBe Some(expectedSize)
+    }
+
+    "return false if EORI is not present" in {
+      val viewModel = createViewModel(Seq(emptySecurityStatementForEori))
+      val result = viewModel.statementRows
+
+      val isEoriPresent = result.nonEmpty
+
+      isEoriPresent mustBe false
+    }
+  }
+
+  "renderPdfLink" should {
+    "return true if PDF size is 1 and true" in {
+      val viewModel = createViewModel(securityStatements)
+      val pdfLink = viewModel.renderPdfLink
+
+      val isSizeOne = pdfLink.size == 1
+
+      isSizeOne mustBe true
+    }
+
+    "render the correct PDF title when no statements" in {
+      val viewModel = createViewModel(securityStatements)
+      val result = viewModel.renderPdfLink
+
+      result mustBe Some(spanComponent("There are no statements available to view.", classes = Some(emptyString), visuallyHidden = false))
```

Comment:
It has become very long line (max is 120 characters), please split over multiple lines. Could be done as below but split it whichever way you can make it readable at the sametime 

      result mustBe Some(
        spanComponent("There are no statements available to view.", classes = Some(emptyString), visuallyHidden = false)
      )